src/index.js

```diff
@@ -0,0 +1,127 @@
+import './style.css';
+import '@fortawesome/fontawesome-free/js/fontawesome.js';
+import '@fortawesome/fontawesome-free/js/solid.js';
+import '@fortawesome/fontawesome-free/js/regular.js';
+import '@fortawesome/fontawesome-free/js/brands.js';
+import statusUpdate from './updateTodos.js';
+import updateTask from './updateTask.js';
+
+function component() {
+  let todos = [];
+  const todoInput = document.querySelector('.todo-input');
+  const todoList = document.querySelector('.todoList');
+
+  function saveLocalTodos(todo) {
+    if (localStorage.getItem('todos') === null) {
+      todos = [];
+    } else {
+      todos = JSON.parse(localStorage.getItem('todos'));
+    }
+    todos.push(todo);
+    localStorage.setItem('todos', JSON.stringify(todos));
+  }
+
+  function getTodos() {
+    if (localStorage.getItem('todos') === null) {
+      todos = [];
+    } else {
+      todos = JSON.parse(localStorage.getItem('todos'));
+    }
+
+    todos.map((todo) => {
+      const newTodo = document.createElement('li');
+      newTodo.classList.add('task');
+      newTodo.setAttribute('id', todo.index);
+      if (todo.completed === true) newTodo.classList.add('completed');
+      const completedCheck = document.createElement('INPUT');
+      completedCheck.setAttribute('type', 'checkbox');
+      completedCheck.setAttribute('id', todo.index);
+      if (todo.completed === true) completedCheck.setAttribute('checked', true);
+      completedCheck.classList.add('check');
+      newTodo.appendChild(completedCheck);
+      const textDescription = document.createElement('INPUT');
+      textDescription.setAttribute('type', 'text');
+      textDescription.setAttribute('value', todo.description);
+      textDescription.classList.add('task-des');
+      textDescription.setAttribute('id', todo.index);
+      newTodo.appendChild(textDescription);
+      const moveButton = document.createElement('button');
+      moveButton.innerHTML = '<i class="fas fa-trash-alt"></i>';
+      moveButton.classList.add('trash-btn');
+      newTodo.appendChild(moveButton);
+      todoList.appendChild(newTodo);
+      return todo;
+    });
+  }
+
+  function addTodo(event) {
+    event.preventDefault();
+    const obj = {
+      description: todoInput.value,
+      index: todos.length + 1,
+      completed: false,
+    };
+    saveLocalTodos(obj);
+    todoInput.value = '';
+    todoList.innerHTML = '';
+    getTodos();
+  }
+
+  function removeLocalTodos(todo) {
+    if (localStorage.getItem('todos') === null) {
+      todos = [];
+    } else {
+      todos = JSON.parse(localStorage.getItem('todos'));
+    }
+    const m = todos.filter((x) => x.index.toString() !== todo.id);
+    const reset = m.map((e, i) => {
+      e.index = i + 1;
+      return e;
+    });
+    todo.remove();
+    localStorage.setItem('todos', JSON.stringify(reset));
+  }
+
+  function deleteCheck(e) {
```

Comment:
This function name seems like it should delete all checked, or something, but from what I understand, it deletes only the selected item. You could change the name to make it more clear.